src/components/Personalization/WallpaperItem.jsx

```diff
@@ -55,11 +61,15 @@ export const WallpaperItem = ({
       {hasCustomWallpaper && (
         <>
           <div
-            className={`${styles['wallpaperOverlay']} u-o-20 u-w-100 u-h-100`}
+            className={cx(styles['wallpaperOverlay'], 'u-o-20 u-w-100 u-h-100')}
           />
           <Circle
-            backgroundColor="var(--white)"
-            className="u-pos-absolute u-top-xs u-right-xs u-o-40"
+            backgroundColor={isMobile ? 'var(--dodgerBlue)' : 'var(--white)'}
```

Comment:
We should not use this color but semantic color. I think primaryColor will do the same?

Reply:
right. Fixed with primary color now